test/graph_rewrites/backend_manager_test.cc

```diff
@@ -159,29 +162,87 @@ TEST(BackendManager, BackendClustering) {
   ASSERT_NE(A_cluster, B_cluster);
 }
 
-// Test Backend Run
-TEST(BackendManager, BackendRun) {
```

Comment:
Removed the BackendRun test, was running end to end and checking for failures in general. Wasnt testing anything specific